.gitignore

```diff
@@ -33,6 +33,7 @@ coverage/
 
 # temporary files
 example/eventLogs/params.json
+example/multiTenancyExampleOnchainPrivacyEventLogs/params.json
```

Comment:
We could replace lines 35 and 36 with something like `**/params.json`. But maybe, for now, we keep it simple :)

Reply:
fair point